cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/CgmesModel.java

```diff
@@ -137,6 +137,8 @@ default PropertyBags fullModel(String cgmesProfile) {
 
     PropertyBags dcTerminalsTP();
 
+    PropertyBags tieFlows();
```

Comment:
Add a default implementation (see for example `topologicalIslands()`)

Reply:
Done